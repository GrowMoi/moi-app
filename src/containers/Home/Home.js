import React from 'react';
import styled from 'styled-components/native';
import { Text, TouchableOpacity } from 'react-native';
import { Actions } from 'react-native-router-flux';

const Home = () => (
  <WrapperScene>
    <TouchableOpacity>
      <Text onPress={Actions.pop}>Press Me!</Text>
    </TouchableOpacity>
  </WrapperScene>
);

const WrapperScene = styled.View`
  flex: 1;
  justify-content: center;
  align-items: center;
`;

export default Home;
