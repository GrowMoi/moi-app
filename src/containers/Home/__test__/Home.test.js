import React from 'react';
import renderer from 'react-test-renderer';

import Home from '../Home';

it('renders without crashing', () => {
  const rendered = renderer.create(<Home />).toJSON();
  expect(rendered).toMatchSnapshot();
  expect(rendered).toBeTruthy();
});
