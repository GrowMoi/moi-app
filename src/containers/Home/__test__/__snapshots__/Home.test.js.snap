// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
      },
      undefined,
    ]
  }
>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    collapsable={undefined}
    hitSlop={undefined}
    isTVSelectable={true}
    nativeID={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
    testID={undefined}
    tvParallaxProperties={undefined}
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      onPress={[Function]}
    >
      Press Me!
    </Text>
  </View>
</View>
`;
